client/src/Service/Client/Internal/ClientApi.php

```diff
@@ -189,4 +189,12 @@ public function getByCaseNumber(string $caseNumber)
     {
         return $this->restClient->get(sprintf(self::GET_CLIENT_BY_CASE_NUMBER_V2, $caseNumber), 'Client');
     }
+
+    /**
+     * @return Client[]
+     */
+    public function searchForClients(array $filters)
+    {
+        return $this->restClient->get('client/get-all?'.http_build_query($filters), 'Client[]');
```

Comment:
Could we get the URL added as a constant in this class so we can give it a logical name?